hack/quick-install.sh

```diff
@@ -5,12 +5,29 @@ set -o xtrace
 
 export CERT_MANAGER_VERSION=v1.9.1
 export KNATIVE_EVENTING_VERSION=v1.14.3
+export JAEGER_OPERATOR_VERSION=v1.57.0
 
+# Production dependencies
 kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml
 kubectl apply -f https://github.com/knative/eventing/releases/download/knative-${KNATIVE_EVENTING_VERSION}/eventing-core.yaml
 kubectl rollout status deployment --namespace=cert-manager --timeout=30s
 kubectl rollout status deployment --namespace=knative-eventing --timeout=30s
 
+# Development dependencies
+# Avoid problems on namespace creation when re-executing this file
+kubectl create namespace observability --dry-run=client -o yaml | kubectl apply -f -
+kubectl apply -f https://github.com/jaegertracing/jaeger-operator/releases/download/${JAEGER_OPERATOR_VERSION}/jaeger-operator.yaml -n observability
+kubectl rollout status deployment --namespace=observability --timeout=30s
+sleep 5  # The jaeger operator webhook its not fast enough :(
+kubectl apply -f - <<EOF
+---
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+  namespace: observability
+EOF
+
 bash cmd/operator/generate.sh
 
 helm install kubearchive charts/kubearchive --create-namespace -n kubearchive \
```

Comment:
This block could use  a logging message as well.

```suggestion
echo "Installing KubeArchive"
helm install kubearchive charts/kubearchive --create-namespace -n kubearchive \
```

Reply:
While theoretically this make sense, practically this gets lost in the output of the script so its not useful. Also, we don't want to redirect the output to somewhere so its more clear if something goes wrong.